tests/acceptance/ENFormCreateCept.php

```diff
@@ -0,0 +1,46 @@
ED_64"},{"id":75694,"name":"volunteer_type","tag":"Not Tagged","property":"NOT_TAGGED_65"},{"id":75695,"name":"volunteer_type_shiptour","tag":"Not Tagged","property":"NOT_TAGGED_66"},{"id":75696,"name":"city_shiptour","tag":"Not Tagged","property":"NOT_TAGGED_67"},{"id":75697,"name":"city_shiptour_openboat","tag":"Not Tagged","property":"NOT_TAGGED_68"},{"id":77983,"name":"testPK","tag":"Not Tagged","property":"NOT_TAGGED_69"},{"id":77990,"name":"birthdayMonth","tag":"Not Tagged","property":"NOT_TAGGED_70"},{"id":79741,"name":"Email address","tag":"Not Tagged","property":"NOT_TAGGED_71"},{"id":80867,"name":"DonorUserRef","tag":"Not Tagged","property":"NOT_TAGGED_72"},{"id":81551,"name":"DDBatchNo","tag":"Not Tagged","property":"NOT_TAGGED_73"},{"id":81606,"name":"PaymentRejectReason","tag":"Not Tagged","property":"NOT_TAGGED_74"},{"id":82578,"name":"Label Text","tag":"Not Tagged","property":"NOT_TAGGED_75"},{"id":88153,"name":"Card_Declined_Imported_SF","tag":"Not Tagged","property":"NOT_TAGGED_76"},{"id":98272,"name":"SFDC Contact ID","tag":"Not Tagged","property":"NOT_TAGGED_77"},{"id":98273,"name":"SFDC Lead ID","tag":"Not Tagged","property":"NOT_TAGGED_78"},{"id":101574,"name":"email_ok_supporter","tag":"Not Tagged","property":"NOT_TAGGED_79"},{"id":101604,"name":"supporter ID Friends","tag":"Not Tagged","property":"NOT_TAGGED_80"},{"id":101605,"name":"date added to friends","tag":"Not Tagged","property":"NOT_TAGGED_81"},{"id":101931,"name":"Date Added to Friends","tag":"Not Tagged","property":"NOT_TAGGED_82"},{"id":102123,"name":"demo","tag":"Not Tagged","property":"NOT_TAGGED_83"},{"id":102168,"name":"owned_PH","tag":"Not Tagged","property":"NOT_TAGGED_84"},{"id":104611,"name":"lead_id","tag":"Not Tagged","property":"NOT_TAGGED_85"},{"id":104612,"name":"Contact_id","tag":"Not Tagged","property":"NOT_TAGGED_86"},{"id":104758,"name":"civi_id","tag":"Not Tagged","property":"NOT_TAGGED_90"},{"id":104759,"name":"supporter_id","tag":"Not Tagged","property":"NOT_TAGGED_91"},{"id":28121,"name":"Email","tag":"Email Address","property":"emailAddress"},{"id":104754,"name":"first_name","tag":"Not Tagged","property":"NOT_TAGGED_87"},{"id":104755,"name":"last_name","tag":"Not Tagged","property":"NOT_TAGGED_88"},{"id":75480,"name":"phone_number","tag":"Not Tagged","property":"NOT_TAGGED_12"},{"id":104756,"name":"supporter_birthday","tag":"Supporter Birthday","property":"dateOfBirth"},{"id":104757,"name":"owned_hungary","tag":"Not Tagged","property":"NOT_TAGGED_89"},{"id":104762,"name":"hu_active_donor","tag":"Not Tagged","property":"NOT_TAGGED_92"},{"id":104763,"name":"hu_recurring_donor","tag":"Not Tagged","property":"NOT_TAGGED_93"},{"id":104764,"name":"card_expiry","tag":"Not Tagged","property":"NOT_TAGGED_94"},{"id":104765,"name":"donor_first_donation_date","tag":"Not Tagged","property":"NOT_TAGGED_95"},{"id":105316,"name":"txn6","tag":"Not Tagged","property":"NOT_TAGGED_96"},{"id":105317,"name":"txn7","tag":"Not Tagged","property":"NOT_TAGGED_97"}]';
```

Comment:
@angtheod can you move the sample api responses to a file and read them from there?
Ideally under _support folder

Could be something like
`_support/plugins/engagingnetworks /api`

Reply:
@kirdia done.